appveyor.yml

```diff
@@ -13,7 +13,7 @@ build_script:
   - conda config --set always_yes yes --set changeps1 no
   - conda config --add channels conda-forge
   - conda update -q conda
-  - conda install pip numpy scipy cython tbb tbb-devel
+  - conda install pip cython matplotlib numpy rowan scipy sympy tbb tbb-devel
```

Comment:
If we're going to introduce rowan as a testing dependency, we should also update other tests (e.g. Cubatic) that implement their own quaternion generation logic, etc.

Reply:
Can you make a separate issue for that? No need to address in this PR.